routes/furniture.js

```diff
@@ -0,0 +1,101 @@
+const express = require('express');
+const { pool, db } = require('../server/db');
+
+const router = express.Router();
+// await pool.query('query')
+
+
+router.get('/', async (req, res) => {
+  try {
+    const furniture = await pool.query('SELECT * FROM furniture');
+    res.status(200).json(furniture.rows);
```

Comment:
Let's pass the result `furniture.rows`  to the `keysToCamel` from `utils.js` so we can standardize the response for the frontend.